src/Entity/DatasetSubmission.php

```diff
@@ -2174,6 +2189,27 @@ public function getKeywords(): ?Collection
         return $this->keywords;
     }
 
+    /**
+     * Stringified getter for Keywords
+     */
+    #[SGroups(['export'])]
+    public function getKeywordsString(): string
+    {
+        $keywordArray = [];
+        foreach ($this->keywords as $keyword) {
+            $label = $keyword->getLabel();
+            if ($keyword->getType() == KeywordType::TYPE_ANZSRC) {
+                $type = KeywordType::TYPE_ANZSRC->value;
+            } elseif ($keyword->getType() == KeywordType::TYPE_GCMD) {
+                $type = KeywordType::TYPE_GCMD->value;
+            } else {
+                $type = 'unknown';
+            }
```

Comment:
Is the if statement really needed? The type should never be null, or something else. But just in case, you can null coalesce.

```suggestion
            $type = $keyword->getType()?->value ?? 'unset';
```